scripts/mac.app/Contents/Info.plist.template

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
 	<key>CFBundleName</key>
-	<string>IGV_@VERSION@</string>
```

Comment:
At one time @VERSION was replaced during the build to the current version, probably from the Jenkins project.  Do we not do that anymore?  That's normally where people go on a Mac to see the version.  They can still get it from the help menu, I think, so maybe this is not a big deal.

Reply:
The Gradle build takes care of this now.  The value is supplied by Jenkins but Gradle does the substitution.  Look for ```filter (ReplaceTokens, ...``` in the build.gradle script.